tool/e2e-tests/test-packs/test-packs.ts

```diff
@@ -1,50 +1,72 @@
-import {ThenableWebDriver, until, WebElement} from "selenium-webdriver"
-import {apiTimeout, defaultWaitTimeout} from "../helpers"
+import { ThenableWebDriver, until } from "selenium-webdriver"
+import { apiTimeout, defaultWaitTimeout } from "../helpers"
 import path from "path"
-import {sendPageTitle} from "../locators"
+import { sendPageTitle } from "../locators"
+import { FileUploadInfo } from "../interfaces/FileUploadInfo.interface"
+
+const TestPackFileUploadInfo: FileUploadInfo = { fileName: "", filePath: "../test-packs/", uploadElementIndex: 0 }
+const FHIRMessageFileUploadInfo: FileUploadInfo = { fileName: "", filePath: "../test-packs/FHIR-messages/", uploadElementIndex: 1 }
 
 export async function loadClinicalFullPrescriberTestPack(driver: ThenableWebDriver): Promise<void> {
-  await loadTestPack(driver, "Full Prescriber Volume Pack.xlsx")
+  const fileInfo: FileUploadInfo = { ...TestPackFileUploadInfo, fileName: "Full Prescriber Volume Pack.xlsx" }
```

Comment:
Could remove these FileUploadInfo types to reduce some visual noise

Reply:
I'm trying to express that fileinfo has to implement the fileInfo interface due to it being passed into loadTestData function as the second param which expects an object that implements  the fileInfo interface.

In my opinion removing the type will increase the chance introducing a bug during future development.

Give me a shout if you want to discuss this in a little more detail?